src/explorer.ts

```diff
@@ -264,8 +267,25 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
         return kubefsUri(this.namespace, this.resourceId, outputFormat);
     }
 
-    getChildren(kubectl: Kubectl, host: Host): vscode.ProviderResult<KubernetesObject[]> {
-        return [];
+    async getChildren(kubectl: Kubectl, host: Host): Promise<KubernetesObject[]> {
+        if (this.kind !== kuberesources.allKinds.pod) {
+            return [];
+        }
+        const result = await kubectl.asJson<Pod>(`get pods ${this.metadata.name} -o json`);
+        if (result.succeeded) {
+            const pod = result.result;
+            let ready = 0;
```

Comment:
extract into a `count(array, predicate)` method, or use `pods.status.containerStatuses.filter((cs) => cs.ready).length`

Reply:
moved to a different PR.